client/cmd.go

```diff
@@ -21,7 +21,7 @@ const ClientContextKey = sdk.ContextKey("client.context")
 // SetCmdClientContextHandler is to be used in a command pre-hook execution to
 // read flags that populate a Context and sets that to the command's Context.
 func SetCmdClientContextHandler(clientCtx Context, cmd *cobra.Command) (err error) {
-	clientCtx, err = ReadPersistentCommandFlags(clientCtx, cmd.Flags())
+	clientCtx, err = ReadPersistentCommandFlags(clientCtx, cmd.Flags(), clientCtx.Codec)
```

Comment:
I assume, i have to make new codec.Codec field on ClientCtx struct, right? cc @amaurym  @robert-zaremba